discovery/kubernetes/pod_test.go

```diff
@@ -128,6 +170,7 @@ func expectedPodTargetGroups(ns string) map[string]*targetgroup.Group {
 					"__meta_kubernetes_pod_container_port_name":     "testport",
 					"__meta_kubernetes_pod_container_port_number":   "9000",
 					"__meta_kubernetes_pod_container_port_protocol": "TCP",
+					"__meta_kubernetes_pod_container_init":          "false",
```

Comment:
```suggestion
					"__meta_kubernetes_pod_container_is_init":          "false",
```
And everywhere else in this file if we agreed on this.

Reply:
ignore this (: Seems like ommiting `is` is what we want. :+1: